### PR TITLE
mustache-interpolation-spacing - ensure that more than one space is invalid when 'always' set

### PR DESCRIPTION
**Pull request by jblotus**

### CODE REVIEW

**Comment by jblotus** [comment]
https://github.com/vuejs/eslint-plugin-vue/issues/214

File: lib/rules/mustache-interpolation-spacing.js [hunk 1]
```diff
@@ -43,6 +43,22 @@ module.exports = {
         const spaces = rightToken.loc.start.column - leftToken.loc.end.column
         const noSpacesFound = spaces === 0
 
+        if (optSpaces && spaces > optSpaces) {
```
Comment (michalsnik) [inline lib/rules/mustache-interpolation-spacing.js]: Wouldn't simple:
```
if(optSpaces === noSpacesFound || (optSpaces && spaces > optSpaces)) {
```
be enough?

Comment (jblotus) [reply lib/rules/mustache-interpolation-spacing.js]: yep - just tested your suggestion. pushing changes now!

**Comment by mysticatea** [comment]
Thank you for contributing.

Hmm, however, I think this is the responsibility of [vue/no-multi-spaces](https://github.com/vuejs/eslint-plugin-vue/blob/master/docs/rules/no-multi-spaces.md) rule.

[This](https://eslint.org/demo/#eyJ0ZXh0IjoiLyplc2xpbnQgXHJcbiAgICB0ZW1wbGF0ZS1jdXJseS1zcGFjaW5nOiBbZXJyb3IsIGFsd2F5c10sXHJcbiAgICBuby1tdWx0aS1zcGFjZXM6IGVycm9yXHJcbiAqL1xyXG5cclxuYCR7ICAgaGVsbG99YFxyXG4iLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOjgsInNvdXJjZVR5cGUiOiJtb2R1bGUiLCJlY21hRmVhdHVyZXMiOnt9fSwicnVsZXMiOnt9LCJlbnYiOnt9fX0=) is the demo about similar rules `template-curly-spacing` and `no-multi-spaces` in core. The `template-curly-spacing` rule reports only missing spaces and `no-multi-spaces` reports too many spaces. It avoids duplicated reports about multi spaces.

**Comment by michalsnik** [comment]
Ah, I didn't know that. In that case we'd rather close this PR as current situation seem to be aligned with core rules. I'm glad you pointed this @mysticatea :)

Sorry for the confusion @jblotus

**Pull request closed without merging**